src/copycutter/text_question.py

```diff
@@ -305,18 +314,32 @@ def parse_copier(self) -> list[Widget]:
                         if "placeholder" in prompt[1]:
                             if "default" in prompt[1]:
                                 # Can have both defaults + placeholders in this case
-                                widgets.append(TextQuestion(prompt[1]["placeholder"], prompt[0], prompt[1]["help"]))
-                                widgets[-1]._input.value = prompt[1]["default"]
+                                q = TextQuestion(
```

Comment:
By assigning this to a variable, we can avoid the `[-1]` on the list (which also loses useful typing info).